### ISSUE TITLE
Need an updated tutorial for seldon serving on GKE

### ISSUE THREAD

**Issue by vivekjhauipath** [SeldonIO/seldon-core#361]
Can you please provide a step by step guide from seldon serving on GKE using Kubeflow? I would ideally like to install Kubeflow using web deployment tool google provides and instlaling Seldon from marketplace. If possible provide a notebook sample that I can run.

**Comment by cliveseldon** [comment]
Seldon is available as part of kubeflow, see [the kubeflow docs](https://www.kubeflow.org/docs/guides/components/seldon/).
If you wish to use the MarketPlace install of Seldon you should be able to run it on a Cluster created by Kubeflow. Is there are reason you wish to use the MarketPlace version especially?
For the latest Seldon, using the Ksonnet or Helm on our main repo would be advisable, but depends how much you want to keep up with the latest updates, see [install docs](https://github.com/SeldonIO/seldon-core/blob/master/docs/install.md)

**Comment by vivekjhauipath** [comment]
Thanks @cliveseldon for your prompt response.
I followed [this guide](https://www.kubeflow.org/docs/started/getting-started-gke/) to install kubeflow on GKE(took the UI route). Post this in jupyter notebook I tried following the [seldon-example](https://github.com/SeldonIO/seldon-core/blob/master/notebooks/ksonnet_examples.ipynb). Here I faced a few issues first was related to permissions and second was unavailability of ks command. Also, this installs seldon inside a ks_app but I want one installation of seldon to be used by multiple apps. Failing to make a headway here I switched to this [e2e github summarization example](https://codelabs.developers.google.com/codelabs/cloud-kubeflow-e2e-gis/index.html#2) (barring cluster creation which I already had because of kubeflow). I used cloud shell for this but every time I tried to do **ks apply gke -c seldon**, cloud shell became unresponsive and disconnected. So, I thought of using Seldon from marketplace because I could easily install it and also it is the first way mentioned in the above mention [install doc](https://github.com/SeldonIO/seldon-core/blob/master/docs/install.md), but I couldn't find how to use it. 
I hope this clarifies it.

**Comment by cliveseldon** [comment]
Hi @vivekjhauipath 

 * For following the [ksonnet example](https://github.com/SeldonIO/seldon-core/blob/master/notebooks/ksonnet_examples.ipynb) you will need to install the prerequisites which include Ksonnet which will install the ks binary.
 * If you want to install a single Seldon Core to control all namespaces you will want single_namespace=false as discussed in the [install docs](https://github.com/SeldonIO/seldon-core/blob/master/docs/install.md)

**Comment by vivekjhauipath** [comment]
Hi @cliveseldon 

- Shouldn't kubeflow installtion included ksonnet (which is present in cli deployment)? So, inside jupyter I used wget to fetch ksonnet and replaced all commands which used ks to point to ks binary. I was unable to add a sym link to ks install directory. I got another issue after this but my guess is that was also related to permissions issue. Do I need to somehow give a different set of permissions to jupyter on GKE?
- Will try this. Thanks.
